pages/tutorials/yaml/species.rst

```diff
@@ -0,0 +1,209 @@
+section name.
+
+Species Name
+------------
+
+The name of a species is given in ``name`` field of a ``species`` entry. Names
+may include almost all printable characters, with the exception of spaces. The
+use of some characters such as ``[``, ``]``, and ``,`` may require that species
+names be enclosed in quotes when written in YAML. Some valid species names given
+in a YAML list include:
+
+.. code:: yaml
+
+    [CH4, methane, argon_2+, "C[CH2]", CH2(singlet), "H2O,l"]
+
+Elemental Composition
+---------------------
+
+The elemental composition of a species is specified as a mapping in the
+``composition`` entry.
+
+For gaseous species, the elemental composition is well-defined, since the
+species represent distinct molecules. For species in solid or liquid solutions,
+or on surfaces, there may be several possible ways of defining the species. For
+example, an aqueous species might be defined with or without including the water
+molecules in the solvation cage surrounding it.
+
+For surface species, it is possible for the ``composition`` mapping to be empty,
+in which case it is composed of nothing, and represents an empty surface site.
+This can also be done to represent vacancies in solids. A charged vacancy can be
+defined to be composed solely of electrons.
+
+The number of atoms of an element must be non-negative, except for the special
+"element" ``E`` that represents an electron.
+
+Examples:
+
+.. code:: yaml
+
+    composition: {C: 1, O: 2}  # carbon dioxide
+    composition: {Ar: 1, E: -2}  # Ar++
+    composition: {Y: 1, Ba: 2, Cu: 3, O: 6.5}  # stoichiometric YBCO
+    composition: {}  # A surface species representing an empty site
+
+
+Thermodynamic Properties
+------------------------
+
+In addition to the thermodynamic model used at the phase level for computing
+properties, parameterizations are usually required for the enthalpy, entropy,
+and specific heat capacities of individual species under standard conditions.
+These parameterizations are provided in the ``thermo`` field of each ``species``
+entry.
+
+The parameterization used to provide this information is specified by the
+``model`` field of the ``thermo`` field. The models available are:
+
+    - :ref:`NASA7 <sec-yaml-nasa7>`
+    - :ref:`NASA9 <sec-yaml-nasa9>`
+    - :ref:`Shomate <sec-yaml-shomate>`
+    - :ref:`constant-cp <sec-yaml-constcp>`
+    - :ref:`piecewise-Gibbs <sec-yaml-piecewise-gibbs>`
+
+The fields used by each model are described and examples provided in the linked
+documentation.
+
+Species Equation of State
+-------------------------
+
+For some phase thermodynamic models, additional equation of state
+parameterizations are needed for each species. This information is provided in
+the ``equation-of-state`` field of each ``species`` entry, with the type of
+parameterization used specified by the ``model`` field of the
+``equation-of-state`` field. The models available are:
+
+    - :ref:`constant-volume <sec-yaml-eos-constant-volume>`
```

Comment:
Would it make sense to clarify some of the acronyms here (e.g. HKFT is not clear). Same goes for other bullet lists of hyperlinks ... in many cases, names are not self-explanatory and it requires multiple clicks to figure out what phase / EOS definitions entail. Is there a better solution?

Reply:
I struggled with this a bit, since I'm trying to avoid duplicating information here that belongs either in the YAML API docs or documentation that is better left for the "Science" section (never mind that this is in many places missing). I ended up adding a very short description for each item in these lists, which hopefully makes this more useful as a tutorial.